tools/swiftdt/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_swift_target_executable(swiftdt
```

Comment:
This is the wrong way to do this. We do not want add_swift_target_executable to be used outside of the stdlib directory. This is in preparation for splitting the stdlib from the main host build.

If you need to be in swift's cmake build, I would suggest putting this tool into ./stdlib/tools.

Otherwise, if you can compile against a just built toolchain, I would create a build-script tool that builds against the just built toolchain.

Reply:
I imagine that you can do the latter since you are just using remote mirrors.